### ISSUE TITLE
purifier support for new uri schemes

### ISSUE THREAD

**Issue by tbba** [ryancramerdesign/ProcessWire#995]
Hi, shouldn't the purifier support new html5 stuff like
href="tel:123"?

https://github.com/robbytaylor/htmlpurifier-tel/tree/master/src

**Comment by tbba** [comment]
Purifier also does not tolerate 2 class names?

**Comment by ryancramerdesign** [comment]
It seems to work ok with 2 class names for me? Can you clarify?

Good idea to add tel: support. But I'm not sure how to add that code... there doesn't seem to be any instructions with that repo?

**Comment by tbba** [comment]
Sorry for not replying earlier. 
I found a reproducible situation:
1. In the PageEditLink Module settings I set this list of classes:
uk-button
uk-button uk-button-primary
uk-button uk-button-success
uk-button uk-button-danger
2. in cKE I make a new link that should be a color- button, in UIKIT that means 2 classes,
eg: uk-button uk-button uk-button-primary
So in Attributes I check 2 classes and hit save.

And mystery mystery: Along the way of saving, only one class remains.

CKE settings:
Formatters: hanna, SupersmartyPants, ThinSpace
ACF yes, Purifier yes
